glide.lock

```diff
@@ -1,8 +1,10 @@
-hash: b22433552234edad060646b980eebe603a7f98b8447c1b81bc831e49791f1fc1
-updated: 2017-02-02T17:05:27.616783744+05:30
+hash: 6c46de8510c27a5b6b56122ac96feda1ad9651b9f6a1ec89bd4ed1bbf272e9fd
+updated: 2017-02-07T11:55:45.327429678+05:30
 imports:
+- name: github.com/asaskevich/govalidator
+  version: 7b3beb6df3c42abd3509abfc3bcacc0fbfb7c877
 - name: github.com/blang/semver
-  version: ff9bd0e2be32754107fc7f37531b104b6219eab8
+  version: 31b736133b98f26d5e078ec9eb591666edfd091f
```

Comment:
It keeps worrying me that adding a new dependency keeps also changing things which are effectively unrelated. This is due to the fact that several of our dependencies are still not specifying versions. At some stage we should do a walk-through the dependencies and see how many we can lock down.